libs/blog/layouts/ui-navigation/src/lib/navigation/navigation.component.html

```diff
@@ -33,7 +34,8 @@
         <li>
           <a
             [attr.data-testid]="item.dataTestId"
-            class="text-al-foreground p-2 font-medium md:p-6"
+            class="p-2 font-medium md:p-6"
+            [ngClass]="!navFooter() ? 'text-al-foreground' : 'text-white'"
```

Comment:
it's probably better to use positive statement in the if/else condition (without !)